CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ add_subdirectory(libtsuba)
 add_subdirectory(libgalois)
 add_subdirectory(libgraph)
 
-if(KATANA_IS_MAIN_PROJECT AND BUILD_TESTING)
+if(KATANA_IS_MAIN_PROJECT AND BUILD_CPP_TESTING)
```

Comment:
`BUILD_TESTING` is a semi-official variable: https://cmake.org/cmake/help/latest/module/CTest.html

but `BUILD_CPP_TESTING` and `BUILD_PYTHON_TESTING` are just our things, so I'd prefer that we give them a `KATANA_` prefix, e.g., `KATANA_LANG_TESTING` in correspondence to `KATANA_LANG_BINDINGS`.

Reply:
Good point. I'll update this when I have the time.